docs/gpa.md

```diff
@@ -180,25 +179,30 @@ This requires the [Covered By](https://github.com/open-contracting-extensions/oc
         <td>the subject-matter of the procurement;</td>
         <td markdown=1>
 
-* 
+* Enter it in `tender/description`
+* If `tender/description` is not empty, enter it after the existing content
 </td>
       </tr>
       <tr>
         <td>VII:3(b)</td>
         <td>the final date for the submission of tenders or, where applicable, any final date for the submission of requests for participation in the procurement or for inclusion on a multi-use list; and</td>
         <td markdown=1>
 
-* 
+* Enter *the final date for the submission of tenders* or, if disclosed, *any final date for the submission of  requests for participation in the procurement or for inclusion on a multi-use list*, in `tender/tenderPeriod/endDate`
```

Comment:
I think in GPA there can be two simultaneous flows. As I understand, the rules around multi-use lists are that suppliers can request inclusion at any time. So, the procedure might already be at the stage of receiving submissions of tenders when a request for inclusion comes in.

The GPA also discusses dates for the submission of tenders without distinguishing between single-stage and multi-stage procedures. We might need to address this and have different mappings for each case. We should review the mapping for VII:2 to make sure it makes sense.

Reply:
I've finally discarded the whole article.